testament/testament.nim

```diff
@@ -307,7 +308,7 @@ proc addResult(r: var TResults, test: TTest, target: TTarget,
       maybeStyledEcho styleBright, expected, "\n"
```

Comment:
showing `expected` is still useful (even with showing the diff) so users can copy paste it
(and showing the `given` is also useful for understanding the output)